std_msgs/Header header
int16 A_speed
int16 B_speed
uint16 A_dur
uint16 B_dur
